db/db_compaction_test.cc

```diff
@@ -6417,10 +6417,10 @@ TEST_P(RoundRobinSubcompactionsAgainstPressureToken, PressureTokenTest) {
   options.level0_file_num_compaction_trigger = 4;
   options.target_file_size_base = kKeysPerBuffer * 1024;
   options.compaction_pri = CompactionPri::kRoundRobin;
-  // Target size is chosen so that filling the level with `kFilesPerLevel` files
-  // will make it oversized by `kNumSubcompactions` files.
-  options.max_bytes_for_level_base =
-      (kFilesPerLevel - kNumSubcompactions) * kKeysPerBuffer * 1024;
+  // Pick a small target level size to pick more files to compact
+  // and trigger subcompaction.
+  // Cannot be too small to cause compaction pressure.
+  options.max_bytes_for_level_base = 40 * kKeysPerBuffer * 1024;
```

Comment:
Can you translate 40 to something related to kFilesPerLevel and kNumSubcompactions just so to avoid hard-coded value?? Maybe kFilesPerLevel - 4* kNumSubcompactions? Also add comment about (1) the actual num subcompaction won't be 4 since it will be constrained by bg compaction number `env_->SetBackgroundThreads(kNumSubcompactions, Env::LOW);` (2) how compaction pressure relates to max_bytes_for_level_base